bidding-impl/src/main/java/com/example/auction/bidding/impl/AuctionState.java

```diff
@@ -10,6 +12,8 @@
 /**
  * The auction state.
  */
+@Value
+@Wither
```

Comment:
I think this `@Wither` added more methods than there where originally. Before this PR, only `withStatus` was available.